config/jail.conf

```diff
@@ -253,6 +253,8 @@ logpath  = %(apache_error_log)s
 [apache-badbots]
 # Ban hosts which agent identifies spammer robots crawling the web
 # for email addresses. The mail outputs are buffered.
+# See filter.d/http-badbots for details
+filter   = http-badbots
```

Comment:
happen someone did get that apache-badbots.local, they would enjoy a regression here... But since this RF is benefitial then may be it would just suffice an "Incompatible changes" moving introduction of http-badbots on top of the changelog pointing to this RF?